### PR TITLE
Tdr 2262 fix login page issue

### PR DESCRIPTION
**Pull request by vimleshtna**
Show a page with a message and a link "Continue to homepage" when the user is already logged in.

### CODE REVIEW

File: themes/tdr/login/info.ftl [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <#import "template.ftl" as layout>
 <#assign backToApplicationText = kcSanitize(msg("backToApplication"))>
+<#assign cnotinueToApplicationText = kcSanitize(msg("continueToApplication"))>
```
Comment (TomJKing) [inline themes/tdr/login/info.ftl:3]: Spelling of variable. Should be: `continueToApplicationText`

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment.

I've created an exploratory testing card for this as well: https://national-archives.atlassian.net/browse/TDR-2386 If you could add some testing instructions to it that would be great.

As this issue is user facing, it would be good to double check the functionality before releasing to production.

File: themes/tdr/login/info.ftl (hunk 1 above)
Comment (vimleshtna) [reply themes/tdr/login/info.ftl:3]: Good catch. I have updated that, thanks.

**Comment by vimleshtna** [comment]
> Looks good. One minor comment.
> 
> I've created an exploratory testing card for this as well: https://national-archives.atlassian.net/browse/TDR-2386 If you could add some testing instructions to it that would be great.
> 
> As this issue is user facing, it would be good to double check the functionality before releasing to production.

Right, I will add some scenarios.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**